src/core_ocean/Registry.xml

```diff
@@ -375,6 +375,22 @@
 					description="If true, the standard GM for the tracer advection and mixing is turned on. This includes the redi portion which is captured in hmix."
 					possible_values=".true. or .false."
 		/>
+		<nml_option name="config_gm_lat_variable_c2" type="logical" default_value=".false." units="NA"
```

Comment:
@jonbob note there are 6 new GM flags added to maint-1.2 with this update.

Reply:
@mark-petersen - I'm only seeing 4? Aren't the others just new arrays?

Reply:
Yes, just 4. My mistake.